pylivetrader/__main__.py

```diff
@@ -82,6 +82,13 @@ def algo_parameters(f):
             default='INFO',
             show_default=True,
             help='The minimum level of log to be written.'),
+        click.option(
+            '-q', '--quantopian-compatible',
```

Comment:
it's a bit vague name. too long but something like quantopian-compatible-pricefilling or something